SPECK/src/main/java/SPEck.java

```diff
@@ -303,85 +383,104 @@ public static String[] parallelGenDataset(int i, int T, double theta, String str
             writeDataset(fileRandom, datasetRandom);
             mining(fileRandom, fileMined, theta);
```

Comment:
Let's call the `fileRandom` variable `fileRandomDataset`, and  the `fileMined` variable `fileRandomFSP`.